website/docs/r/schema_template_contract_service_graph.html.markdown

```diff
@@ -15,8 +15,8 @@ Manages MSO Schema Template Contract service graph.
 ```hcl
 
 resource "mso_schema_template_contract_service_graph" "one" {
-  schema_id               = "5f11b0e22c00001c4a812a2a"
-  site_id                 = "5c7c95b25100008f01c1ee3c"
+  schema_id               = mso_schema.schema1.id
+  site_id                 = mso_site.site1.id
   template_name           = "Template1"
   contract_name           = "UntitledContract1"
```

Comment:
should point to mso_schema_template_contract or mso_schema_template_contract_filter

Reply:
ok